model/pull_request.go

```diff
@@ -28,4 +28,5 @@ type PullRequest struct {
 	URL                 string
 	CreatedAt           time.Time
 	MaintainerCanModify bool
+	Merged              bool
```

Comment:
Also, you need to add:

- SQL migration file [like this one](https://github.com/mattermost/mattermost-mattermod/pull/164/files#diff-9e8f48247685706a15181d89c05243beR1) that is going to be executed by @metanerd . You can use a default value [to false](https://github.com/google/go-github/blob/master/github/github-accessors.go#L8986) but I rather prefer to leave it null, use the NULLXXX field and then make the `Ticket` updates it (it's going to do it anyway) 
- Given that the flag comes from the response you should also add something like [this](https://github.com/mattermost/mattermost-mattermod/pull/164/files#diff-fec73530a7655cd7c507f18fc6ed23cdR242)